packages/web-components/src/components/rux-tabs/test/tabs.spec.ts

```diff
@@ -297,7 +338,6 @@ test.describe('Multiple tabs on same page', () => {
         await expect(topContent2).toBeVisible()
         await expect(bottomContent).toBeVisible()
     })
-    //TODO: Replace instances of e.evaulate here with the .hasAttribute
```

Comment:
Refactored tests while I was in there

Reply:
Good call on the test refactoring. Not having to use that evaluate stuff to check for attributes is great!